src/compile.ts

```diff
@@ -11,6 +11,7 @@ const compilerModule = require.resolve('jsii/bin/jsii');
 export async function compile(workdir: string, options: Options) {
   const args = [ '--silence-warnings', 'reserved-word' ];
   const entrypoint = options.entrypoint ?? 'index.ts';
+  const packageName = options.packageName?.replace(/\./g, '').replace(/\//g, '') ?? 'generated';
```

Comment:
Sanitizing input here because of an issue in JSII. If a packageName such as `jenkins.io/jenkins` is passed in, JSII will throw:

https://github.com/aws/jsii/blob/master/packages/jsii/lib/validator.ts#L203

Converting that to just `jenkinsiojenkins` seems to fix the issue. It's not pretty, but better than just a random ID and I don't expect people to look too closely into the imports.